include/swift/ABI/Task.h

```diff
@@ -485,13 +490,24 @@ class AsyncTask : public Job {
     return reinterpret_cast<AsyncTask *&>(
         SchedulerPrivate[NextWaitingTaskIndex]);
   }
+
+  /// Get the next non-zero Task ID.
+  uint32_t GetNextTaskId() {
+    static std::atomic_uint32_t Id;
+    uint32_t Next = Id.fetch_add(1, std::memory_order_relaxed);
+    if (Next == 0) Next = Id.fetch_add(1, std::memory_order_relaxed);
```

Comment:
Maybe initialize it to 1 just so we don't trip this special case the first time?  Minor, but still.